src/main/java/gift/domain/member/entity/Member.java

```diff
@@ -34,6 +36,9 @@ public class Member {
     @Enumerated(value = EnumType.STRING)
     private AuthProvider authProvider;
 
+    @ColumnDefault("0")
+    private int point;
```

Comment:
```suggestion
    @ColumnDefault("0")
    private int point = 0;
```
객체 관점에서도 default를 이렇게 지정할수도 있을거 같아요.

Reply:
기본값이라 별도로 지정하지 않았는데 명시해주는 것도 직관적으로 인식하는 데 도움이 될 것 같습니다!